src/random_test.cpp

```diff
@@ -51,11 +51,27 @@ static std::string get_result(std::string sql, Thd1 *thd) {
   return result;
 }
 
-/* return the db branch */
-static std::string db_branch() {
-  std::string branch = mysql_get_client_info();
-  branch = branch.substr(0, 3); // 8.0 or 5.7
-  return branch;
+/* return PS version in number format
```

Comment:
replace ps with ms.

Reply:
We are using it with Percona software and also it is buit within Percona. Should we not use PS?

Reply:
Replaced "ps" with the word "server"